saleor/plugins/webhook/utils.py

```diff
@@ -293,3 +320,27 @@ def delivery_update(delivery: "EventDelivery", status: str):
 def clear_successful_delivery(delivery: "EventDelivery"):
     if delivery.status == EventDeliveryStatus.SUCCESS:
         delivery.delete()
+
+
+WEBHOOK_TAX_CODES_CACHE_KEY = "webhook_tax_codes"
+DEFAULT_TAX_CODE = "SA0000"
+DEFAULT_TAX_DESCRIPTION = "Unmapped Product/Product Type"
+
+
+def get_current_tax_app() -> Optional[App]:
+    """Return currently used tax app or None, if there aren't any."""
```

Comment:
What in situations that we have 3 different apps for taxes? 
The last app can be not active.

Reply:
Assigning a tax codes doesn't support more than one enabled tax App. `is_active` is checked in for_event_type.